Looks like @types/history is not valid

I've got errors after [`@types/history`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/types-2.0/history/index.d.ts) is installed:

```
node_modules/@types/history/index.d.ts(120,42): error TS2307: Cannot find module 'history/lib/createBrowserHistory'.
node_modules/@types/history/index.d.ts(121,46): error TS2307: Cannot find module 'history/lib/createHashHistory'.
node_modules/@types/history/index.d.ts(122,48): error TS2307: Cannot find module 'history/lib/createMemoryHistory'.
node_modules/@types/history/index.d.ts(123,43): error TS2307: Cannot find module 'history/lib/createLocation'.
node_modules/@types/history/index.d.ts(124,40): error TS2307: Cannot find module 'history/lib/useBasename'.
node_modules/@types/history/index.d.ts(125,44): error TS2307: Cannot find module 'history/lib/useBeforeUnload'.
node_modules/@types/history/index.d.ts(126,39): error TS2307: Cannot find module 'history/lib/useQueries'.
node_modules/@types/history/index.d.ts(127,26): error TS2307: Cannot find module 'history/lib/actions'.
```

Looks like `lib` folder from DefinitelyTyped repository is not included into `@types/history` module.

--- a/src/lib/definition-parser.ts
+++ b/src/lib/definition-parser.ts
@@ -242,9 +242,10 @@ function imports(src: ts.SourceFile): string[] {
 
 	for (const node of src.statements) {
 		switch (node.kind) {
-			case ts.SyntaxKind.ImportDeclaration: {
-				const decl = node as ts.ImportDeclaration;
-				if (decl.moduleSpecifier.kind === ts.SyntaxKind.StringLiteral) {
+			case ts.SyntaxKind.ImportDeclaration:
+			case ts.SyntaxKind.ExportDeclaration: {
+				const decl = node as ts.ImportDeclaration | ts.ExportDeclaration;
+				if (decl.moduleSpecifier && decl.moduleSpecifier.kind === ts.SyntaxKind.StringLiteral) {
 					out.push(stripQuotes(decl.moduleSpecifier.getText()));
 				}
 				break;

